### PR TITLE
Update params for interopnet for fvm

### PR DESCRIPTION
**Pull request by travisperson**
- Sets BlockDelay to `5s`
- Sets PropagationDelay `1s` (discussed 500ms, but lotus doesn't use a duration for some reason).
- Sets Network genesis to network version 14, with an ohsnap upgrade.
- Support 512MB, 32GB, 64GB sectors.
- Set InsecurePoStValidation

### CODE REVIEW

File: build/params_interop.go [hunk 1]
```diff
@@ -47,21 +47,22 @@ var UpgradeTurboHeight = abi.ChainEpoch(-15)
...
-var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
+var UpgradeOhSnapHeight = abi.ChainEpoch(300)
 
 var DrandSchedule = map[abi.ChainEpoch]DrandEnum{
 	0: DrandMainnet,
 }
 
 func init() {
+	policy.SetMinVerifiedDealSize(abi.NewStoragePower(256))
+	policy.SetPreCommitChallengeDelay(abi.ChainEpoch(10))
```
Comment (travisperson) [inline build/params_interop.go:58]: Do we want to remove these?

Comment (arajasek) [reply build/params_interop.go:58]: Nah, leave em in, good for testing

File: build/params_interop.go [hunk 2]
```diff
@@ -47,21 +47,24 @@ var UpgradeTurboHeight = abi.ChainEpoch(-15)
 
 var UpgradeHyperdriveHeight = abi.ChainEpoch(-16)
 var UpgradeChocolateHeight = abi.ChainEpoch(-17)
-var UpgradeOhSnapHeight = abi.ChainEpoch(-18)
+var UpgradeOhSnapHeight = abi.ChainEpoch(300)
```
Comment (arajasek) [inline build/params_interop.go:50]: This needs to stay disabled -- the FVM can't support v15 (the SnapDeals upgrade yet)

**Review by arajasek — CHANGES_REQUESTED** [review]

**Comment by jennijuju** [comment]
don't forget to update the PR title according to the pr template once the PR is ready to review!

**Review by arajasek — APPROVED** [review]

**Pull request merged**